runtime/doc/eval.txt

```diff
@@ -4457,6 +4457,9 @@ getbufinfo([{dict}])
 			changed		TRUE if the buffer is modified.
 			changedtick	number of changes made to the buffer.
 			hidden		TRUE if the buffer is hidden.
+			last_used	time in seconds when the buffer was
+					last used.
+					{only with the |+viminfo| feature}
```

Comment:
I would remove the underscore. Because `changedtick` doesn't have one, and for parallel with `list:lastused`.

Reply:
Agree